### DISCUSSION TITLE
`[@types/braintree-web] Type missing`

### DISCUSSION THREAD

**Discussion by wangyq1996** [DefinitelyTyped/DefinitelyTyped#62892] [Issues with a @types package]
General information
    SDK version: 3.75.22 (`braintree-web: 3.88.1`)

Issue description
    I need to set `allowDesktop: true` & `paymentMethodUsage: "multi-use"` to meet out business requirements. However, I can not found these two types definition in @types/braintree-web v 3.75.22. Is there anything I need to pay attentions on? Thx.

[a comment by an automated account is not shown]

**Comment by wangyq1996** [comment]
P.s. The target function is `venmo.create`, thx.

**Reply by Chip12345** [reply]
Ok

**Reply by wangyq1996** [reply]
> Ok

Hi, is there any updates on this?

**Reply by Semigradsky** [reply]
I can update typings on weekend.

**Reply by wangyq1996** [reply]
> I can update typings on weekend.

Cool, appreciate.
